### ISSUE TITLE
Regular social remote chat

### ISSUE THREAD

**Issue by mrocklin** [dask/community#142]
Hi All, 

In order to promote more social bonds between folks in the Dask community, and to provide a more welcoming and less technical entry point, we might consider a couple of regular, half-hour, remote meetings similar to coffee/tea breaks in an office environment  This would be a time to chat about the latest news, kids, pets, and so on.  

### History

Pangeo did this at the start of COVID-19 quarantine and it worked well.

@jsignell did this among maintainers for a while and it worked well.  In particular @GenevieveBuckley mentioned that this was helpful when onboarding, but now with daylight savings time the times don't work out particularly well.

### Times

I might suggest two weekly 30-minute times, one that is America/EMEA friendly, and one that is APAC/America friendly.

**Comment by jrbourbeau** [comment]
+1

**Comment by mrocklin** [comment]
I would like to attend at least the first few.  I have a standing conflict
on US Monday.

**Comment by jacobtomlinson** [comment]
+1

**Comment by douglasdavis** [comment]
Matthew Rocklin ***@***.***> writes:

> On Tue, Mar 30, 2021 at 6:16 PM Genevieve Buckley ***@***.***>
> wrote:
>
>> +1
>>
>> For the APAC/America friendly time, I suggest 9am AEST / 4pm PDT / 6pm
>> CDT. Does anyone have a preference for which day of the week?

> I would like to attend at least the first few. I have a standing
> conflict
> on US Monday.

+1 for me, I'd be able to join 7pm EDT Tues-Fri (would also be
interested in whatever time targets Americas/EMEA).

**Comment by mmccarty** [comment]
+1

**Comment by TomAugspurger** [comment]
I made some recurring invitations at https://calendar.google.com/calendar/embed?src=4l0vts0c1cgdbq5jhcogj55sfs%40group.calendar.google.com. Can folks check that I got the timezones correct?

I think there's another DST transition next week. It seems that Google just gave up on the 4/12 meeting. I can make one manually for that time if the others look correct.

**Comment by lesteve** [comment]
I'd be keen to join from time to time. Maybe I missed it  but I could not find the connection details in the Google calendar. Maybe they are the same as the monthly Dask meeting ?

**Comment by jsignell** [comment]
I imagine it'd be in https://whereby.com/dask-dev

**Comment by TomAugspurger** [comment]
I added the link to https://whereby.com/dask-dev in the calendar.

**Comment by GenevieveBuckley** [comment]
Can you add the link to both the events in the calendar @TomAugspurger? It's still missing from the other timeslot

**Comment by TomAugspurger** [comment]
Strange, should be fixed now. I also shortened that one to be 30 minutes. I must have made it an hour initially on accident.

**Comment by GenevieveBuckley** [comment]
Thanks @TomAugspurger. The time seems pretty odd, am I missing context from a previous conversation?
It's an hour and a half later than I expected (ok for me, probably less so for US folk), and the day itself is strange (I expected to see a US Tuesday/Wednesday/Thursday, equivalent to our Aussie Wed/Thurs/Fri). I don't imagine anyone in the US would be keen to chat about Dask on their Sunday afternoon/evening.

**Comment by TomAugspurger** [comment]
I think that I or Google was confused by time zones, possibly complicated by the DST transition that recently took place.

I was going off the discussion at https://docs.google.com/document/d/1UqNAP87a56ERH_xkQsS5Q_0PKYybd5Lj2WANy_hRzI0/edit#heading=h.vpveq7guu8ws (first "Larger Discussion Item") where we talked about Tuesdays at 9:00 AEST and Thursdays at 9:00 US Eastern. Hopefully that's what's on the calendar now, but let me know if not!

**Comment by GenevieveBuckley** [comment]
The time looks correct now, but Matt has a [conflict](https://github.com/dask/community/issues/142#issuecomment-810640357) on US Mondays / Aussie Tuesdays. I'd prefer we move it a day later to accommodate that (unless you are already accommodating other people's schedules that I don't know about).

**Comment by mrocklin** [comment]
Yeah, I think that my intention was to propose US-Tuesday at around 9am Australian.  I can make either time (although Monday is harder for me).  In general my guess is that Monday social hours that are late in the day are less likely to be attended by US based folks than they would be on Tuesday.  It's a bit nicer (I think) to have social time a bit separated from the weekend, when ones social reserves are maybe already somewhat full.

**Comment by crusaderky** [comment]
+1

**Comment by GenevieveBuckley** [comment]
@TomAugspurger could you shift the America/Oceania meeting to a day later? (I don't have access to that calendar, otherwise I wouldn't be bugging you about it)

**Comment by TomAugspurger** [comment]
Done!

**Comment by GenevieveBuckley** [comment]
A reminder that the next Dask social chat is in <24 hours. See you at https://whereby.com/dask-dev

Calendar link: https://calendar.google.com/calendar/u/0/embed?src=4l0vts0c1cgdbq5jhcogj55sfs@group.calendar.google.com

**Comment by mrocklin** [comment]
Thanks for the reminder.  I'll see folks there!

**Comment by GenevieveBuckley** [comment]
The next Dask social chat is on again in less than 24 hours. Come join us at https://whereby.com/dask-dev at this time every week: https://everytimezone.com/s/ffaf2586

(You can see the full dask calendar of events here: https://docs.dask.org/en/latest/support.html )

**Comment by mrocklin** [comment]
I'll be there!

**Comment by GenevieveBuckley** [comment]
The next Dask social chat is on again in less than 24 hours. Come join us at https://whereby.com/dask-dev at this time every week: https://everytimezone.com/s/ffaf2586

(You can see the full dask calendar of events here: https://docs.dask.org/en/latest/support.html )

**Comment by GenevieveBuckley** [comment]
The next Dask social chat is on again in less than 24 hours. Come join us at https://whereby.com/dask-dev at this time every week: https://everytimezone.com/s/ffaf2586

(You can see the full dask calendar of events here: https://docs.dask.org/en/latest/support.html )

**Comment by GenevieveBuckley** [comment]
And a warning - the next regular Americas/Europe Dask social chat at [this time](https://everytimezone.com/s/631358a9) will not run, because it is on at the same time as the Dask summit plenary session & fireside chat. So you should go to that instead :)

**Comment by GenevieveBuckley** [comment]
There has been very low attendance at the Asia-Pacific friendly time, so I propose we reduce the frequency to once a month (unless there are any objections).

**Comment by GenevieveBuckley** [comment]
A reminder that the next Dask social chat is in <24 hours. See you at https://whereby.com/dask-dev

Calendar link: https://calendar.google.com/calendar/u/0/embed?src=4l0vts0c1cgdbq5jhcogj55sfs@group.calendar.google.com

**Comment by GenevieveBuckley** [comment]
A reminder that the next Dask social chat is in <24 hours ([click to see when in your timezone](https://everytimezone.com/s/1a43edc7)). Join us at  https://whereby.com/dask-dev

Calendar link: https://calendar.google.com/calendar/u/0/embed?src=4l0vts0c1cgdbq5jhcogj55sfs@group.calendar.google.com

**Comment by GenevieveBuckley** [comment]
The next Dask social chat is in <24 hours, at 5pm Eastern Standard Time ([click to see when in your timezone](https://everytimezone.com/s/ab479c0d)). This is probably the last social chat we'll hold for this particular timeslot, so come say hi! Join us at https://whereby.com

**Comment by GenevieveBuckley** [comment]
Sorry, I messed up the link earlier. We're at https://whereby.com/dask-dev right now :smile:
